config/types/files.go

```diff
@@ -70,123 +69,120 @@ type Link struct {
 	User       FileUser  `yaml:"user"`
 	Group      FileGroup `yaml:"group"`
 	Hard       bool      `yaml:"hard"`
-	Target     string    `json:"target"`
+	Target     string    `yaml:"target"`
 }
 
 func init() {
-	register2_0(convertFiles)
-}
-
-func convertFiles(in Config, ast validate.AstNode, out ignTypes.Config, platform string) (ignTypes.Config, report.Report, validate.AstNode) {
-	r := report.Report{}
-	files_node, _ := getNodeChildPath(ast, "storage", "files")
-	for i, file := range in.Storage.Files {
-		fmt.Printf("converting file %q\n", file.Path)
-		file_node, _ := getNodeChild(files_node, i)
-		newFile := ignTypes.File{
-			Node: ignTypes.Node{
-				Filesystem: file.Filesystem,
-				Path:       file.Path,
-				User: ignTypes.NodeUser{
-					ID:   file.User.Id,
-					Name: file.User.Name,
+	register2_0(func(in Config, ast validate.AstNode, out ignTypes.Config, platform string) (ignTypes.Config, report.Report, validate.AstNode) {
```

Comment:
wasn't this changed in an earlier commit?

Reply:
Oh I didn't intend for this to make it into the PR. Ignore this, I'll rip it out.